docs/operations/admin.rst

```diff
@@ -35,6 +35,9 @@ modify different aspects of the server.
     weight, Integer, Load balancing weight (1-100)
     zone, String, Service zone
     canary, Boolean, Whether the host is a canary
+    success_rate, Double, "Request success rate (0-100). -1 if there was not enough
```

Comment:
Link to some docs here (or write it here) that describe what interval this is over.